spring-core/src/main/java/service/EventService.java

```diff
@@ -0,0 +1,40 @@
+package service;
+
+import dao.EventDao;
+import model.Event;
+
+import java.util.Date;
+import java.util.List;
+
+public final class EventService {
```

Comment:
for all 3 service please remove final here,  because you already have declaration with beans prototypes it's redundant

Reply:
when I'm declaring final for class, I also say that I don't want to extend this class in future

Reply:
the service cannot be extended, since this is a pattern and should not be broken, so please delete it